python/tank/platform/engine.py

```diff
@@ -882,7 +933,7 @@ def _initialize_dark_look_and_feel(self):
             fh.close()
             
             # set the std selection bg color to be 'shotgun blue'
-            self._dark_palette.setBrush(QtGui.QPalette.Highlight, QtGui.QBrush(QtGui.QColor("#30A7E3")))
+            self._dark_palette.setBrush(QtGui.QPalette.Highlight, QtGui.QBrush(QtGui.QColor(constants.SG_HIGHLIGHT_COLOR)))
             self._dark_palette.setBrush(QtGui.QPalette.HighlightedText, QtGui.QBrush(QtGui.QColor("#FFFFFF")))
```

Comment:
I was discussing this with @sgsteph , trying to keep it as simple as possible. But there is a Shotgun "light gray" that we use, that I think make sense adding (`"SG_FOREGROUND_COLOR": "#c8c8c8"`). The `#FFF` is a qt stylesheet detail really - something we need to patch because of QT being a bit iffy - just the color change that happens when text is highlighted (see examples here: https://github.com/shotgunsoftware/tk-core/pull/80 ), so not sure it's worth putting this in as a constant.